apps/shared/account/account.component.html

```diff
@@ -0,0 +1,46 @@
+<img (click)="open()" referrerpolicy=“no-referrer” [src]="account.avatarUrl">
+
+<ng-template #menuContent>
+  <div class="account-menu-container mat-elevation-z8 mat-menu-panel">
+    <img referrerpolicy=“no-referrer” class="user-photo" [src]="account.avatarUrl">
+
+    <div class="title-row">
+      <span class="title">{{ account.displayName || 'Welcome!' }}</span>
+      <button class="expand-button" mat-icon-button (click)="providerPanel.toggle()" *ngIf="account.isLoggedIn">
+        <mat-icon>{{providerPanel.expanded ? 'expand_less' : 'expand_more'}}</mat-icon>
+      </button>
+    </div>
+    <ng-container cdkAccordionItem #providerPanel="cdkAccordionItem">
+      <div *ngIf="providerPanel.expanded" class="provider-panel">
+        <div class="provider-row">
+          <img class="provider-icon" src="http://google.com/favicon.ico" />
+          <span class="provider-email">{{ account.googleInfo?.email || "Unknown" }}</span>
+        </div>
+        <div class="provider-row">
+          <img class="provider-icon" src="http://github.com/favicon.ico" />
+          <span class="provider-email">{{ account.githubInfo?.email || "Account Not Linked" }}</span>
+          <button *ngIf="account.githubInfo === null" mat-icon-button class="link-provider-account"
+            (click)="account.linkWithGithub()">
+            <mat-icon>add_link</mat-icon>
+          </button>
+          <button *ngIf="account.githubInfo" mat-icon-button class="link-provider-account"
+            (click)="account.unlinkFromGithub()">
+            <mat-icon>link_off</mat-icon>
+          </button>
+        </div>
+      </div>
+    </ng-container>
+
+    <div class="projected-content">
+      <ng-content></ng-content>
```

Comment:
Is there any information on what the projected content is used for? It's not quite clear to me (yet 😄)